tests/catboost/test_catboost_model_export.py

```diff
@@ -17,7 +17,7 @@
 import mlflow.pyfunc.scoring_server as pyfunc_scoring_server
 from mlflow import pyfunc
 from mlflow.models import Model, ModelSignature
-from mlflow.models.utils import _read_example
+from mlflow.models.utils import _read_example, load_serving_example_from_uri
```

Comment:
Can we rename `load_serving_example_from_uri` to `load_serving_example` as it also accepts a path?